File: burdine-brian/array-plus-array.js
/*
  Psuedocode
    START
    READ arrays of integers arr1 and arr2
    INIT sum as 0
    FOR every item in arr1
      CALCULATE sum = sum + item of arr1
    FOR every item in arr2
      CALCULATE sum = sum + item of arr2
    SHOW sum
*/

function arrayPlusArray(arr1, arr2) {
    let sum = 0;
    //add numbers in arr1
    for (num of arr1) {
      sum += num;
    }
    //add numbers in arr2
    for (num of arr2) {
      sum += num;
    }
    console.log(sum);
    return sum; //return total
  }

File: burdine-brian/array-plus-array.py
def array_plus_array(arr1,arr2):
    arr3 = arr1 + arr2
    sum = 0
    for num in arr3:
        sum += num
    print("Array elements: ", arr3, "Sum: ", sum)
    return sum

# Pseudocode
# Start array_plus_array
# get arr1, arr2
# combine the arrays and store it in arr3
# init sum as 0
# for every number in arr3
#   add number to sum
# endfor
# return sum
# end array_plus_array